content/blog/2016/2016-08-24-jenkins-world-2016-festivities.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Jenkins World 2016: Festivities'
+tags:
+- jenkins world
+- conference
+author: alyssat
+---
+image:jenkins.io/content/images/conferences/Jenkins-World_125x125.png[role='right']
+ At link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2016] on September 14-15, stop by the "Open Source Hub", located in the Partner Expo hall at the Santa Clara Convention Center in Santa Clara, CA. The Open Source Hub will have many Jenkins contributors, committers, JAM leaders, and link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Board[officers from the governance board] under one roof, so there will be plenty of knowledge and talents on hand to share. We hope you’ll join in on the festivities.
+ 
+== Ask the Experts 
+The setup is waiting for you: white boards, monitors and lots of brain power to help answer those Jenkins questions that’s been keeping you up at night. Jenkins experts can help with beginner questions to the more advanced ones. All you need to do is bring your laptop, your questions and get your questions answered by the experts. Let the experts help solve your Jeproblems.
```

Comment:
"All you need to do is bring your laptop and your questions; the experts will help answer them!"